app/snippets/content_templates/glossaryTerm.json

```diff
@@ -0,0 +1,10 @@
+{
+	"encoding": "markdown",
+	"value": "_Enter term here_",
```

Comment:
This doesn't quite look right as a placeholder, e.g. it actually sets it as the glossary term if you leave it (see other comment for suggestions)

Reply:
Done in 0e96f78624bfac65c3b441d04566275f5259580f.